app/models/portfolio_item.rb

```diff
@@ -29,6 +30,13 @@ def metadata
     ancillary_metadata.metadata_attributes.merge('user_capabilities' => user_capabilities, 'statistics' => statistics_metadata)
   end
 
+  def validate_deletable
+    unless deletable?
+      errors.add(:base, "cannot be deleted because it is used by some order processes")
```

Comment:
I think this is fine for this PR, but I can imagine users being potentially confused about *which* order processes are currently using this portfolio item. I can definitely see a customer having 50+ order processes and not knowing which one(s) this portfolio item belongs to. It would be very annoying to have to sift through all of them. I think we should enhance this error message (in a separate PR) to include the name of the order processes that are holding back this deletion.

Reply:
@eclarizio If in the case you mentioned, the changed error message will include 50+ names, is this what we want?

Reply:
@hsong-rh Well, my example was if they had 50+ order processes, but say the portfolio item was only attached to 3 or 4 of them. The list would only be 3 or 4 names long. You're right though, if it's attached to more, that list will be really long. Probably something we can confer with Susan on, I'll tag her in the JIRA story.